src/shared/test_util/test_util.cpp

```diff
@@ -14,4 +14,24 @@ namespace TestUtil
                   << "[" << robot_log.file_name << ":" << robot_log.line_number
                   << "]: " << robot_log.log_msg;
     }
+
+    RobotConstants_t createMockRobotConstants()
```

Comment:
nit; for a few functions we've added in this PR, `create` is a bit redundant, this could probably just be `mockRobotConstants` for example.

Reply:
create avoids problems with starting function names with numbers... is that a good reason?